RawCMS.Library/Core/Helpers/ReflectionHelper.cs

```diff
@@ -170,14 +170,19 @@ public List<Type> GetImplementors(Type t, List<Assembly> bundledAssemblies)
             {
                 // _logger.LogDebug("loading from" + assembly.FullName);
                 Type[] types = assembly.GetTypes();
+                string[] excludedTypeNames = GetExcludedTypeNames();
+
                 foreach (Type type in types)
                 {
                     try
                     {
                         if ((t.IsAssignableFrom(type) || (type.GetInterfaces().Any(i => i.IsGenericType && i.GetGenericTypeDefinition() == t)))
                             && !type.IsAbstract && !type.IsInterface && !type.IsGenericType)
                         {
-                            result.Add(type);
+                            if (!excludedTypeNames.Contains(type.Name)) 
```

Comment:
Hi @TimurMirzaiev, thank you for contributing. 
This fix must be done on the UI where the dropdown is loaded.  The acceptance test for this task is to open the same window of the related task and don't see the two lines.

I will close this PR, for any more info about the task please comment in the task thread.